WebRoot/css/zm.css

```diff
@@ -6529,7 +6487,7 @@ a.ZmLinkDisabled {
 }
 
 .ZmMailAttachmentIcon,
-.ZmMailAttachmentIcon .svg-icon {
+.ZmMailAttachmentIcon svg.svg-icon {
```

Comment:
I think we are moving only direct `.svg-icon.*` kind of rules, which will be common for all SVG's.
However, some SVG in some containers may need some customization, so I think it's better if keep such rules together for logical grouping of rules.